ssp

```diff
@@ -909,16 +942,16 @@ sub version_compare {
 
 sub cpanel_version_is {
     my ( $mode, $ver ) = @_;
-    return if !defined($cpanel_version);
-    return if $cpanel_version eq "unknown";
-    return version_compare( $cpanel_version, $mode, $ver );
+    return if !defined($CPANEL_VERSION);
+    return if $CPANEL_VERSION eq "unknown";
```

Comment:
We could lay off the quotes here as nothing is interpolating.

Reply:
Large commit but clear cut.  I don't have any real advise of value worth changing here so I am approving.